asyncsleepiq/fuzion/foundation.py

```diff
@@ -17,25 +17,25 @@
     Speed,
 )
 from ..foundation import SleepIQFoundation
-from ..preset import SleepIQPreset
+from .preset import SleepIQFuzionPreset

```

Comment:
@kbickar Looking for feedback here - I think since this is the fuzion foundation, it should be using the fuzion preset (like we do for `actuator` and `light` below) and the only parent import should be for the foundation itself.